src/lib.rs

```diff
@@ -596,11 +596,11 @@ impl NetmapDescriptor {
         return None;
     }
 
-    #[allow(dead_code)]
     pub fn get_fd(&self) -> i32 {
         self.desc.get().fd
     }
 
+    #[deprecated]
```

Comment:
I disagree with your reasoning here. For example, in my project I give a netmap descriptor for a ring to each thread and call poll(). But yes, it kinda feels out of place here.